pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1212,6 +1212,37 @@ func (c *PodAffinityChecker) podMatchesPodAffinityTerms(pod, targetPod *v1.Pod,
 	return true, true, nil
 }
 
+// podMatchesAnyPodAffinityTerms checks if the "targetPod" matches any of the given "terms"
+// of the "pod" on the given "nodeInfo".Node(). It returns three values: 1) whether
+// targetPod matches any of the terms and their topologies, 2) whether targetPod
+// matches any of the terms label selector and namespaces (AKA term properties),
+// 3) any error.
+func (c *PodAffinityChecker) podMatchesAnyPodAffinityTerm(pod, targetPod *v1.Pod, nodeInfo *schedulercache.NodeInfo, terms []v1.PodAffinityTerm) (bool, bool, error) {
```

Comment:
considering this function is quite similar with `podMatchesPodAffinityTerms`, it'd nice to combine them to avoid duplicated code.